### PR TITLE
Add gender() to name library

### PR DESCRIPTION
**Pull request by damonpam**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:https://github.com/Marak/faker.js/blob/22fc11df449763e3902dc0964125c35f97842fb3/lib/name.js#L143

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## faker ([<kbd>unpkg</kbd>](https://unpkg.com/browse/faker@latest/))
was missing the following properties:
1. locales
1. localeFallback
1. definitions
1. unique
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6393a06decd70584d96ee493bc7e25e10ce9553e
</p>

File: types/faker/index.d.ts [hunk 1]
```diff
@@ -204,6 +205,7 @@ declare namespace Faker {
             jobDescriptor(): string;
             jobArea(): string;
             jobType(): string;
+            gender(): string;
```
Comment (TastefulElk) [inline types/faker/index.d.ts:208]: Nitpick but could you please add it between `jobTitle` and `prefix` so we use the same order as the Faker lib - making it easier to spot missing properties in the future? 👍

**Review by TastefulElk — CHANGES_REQUESTED** [review]

**Comment by TastefulElk** [comment]
Thank you for fixing that! 🏅

**Review by TastefulElk — APPROVED** [review]

**Comment by damonpam** [comment]
Ready to merge

**Pull request merged**